src/components/Card/Card.stories.mdx

```diff
@@ -0,0 +1,529 @@
+
+<Meta
+  title={getCategory("Card")}
+  component={Card}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=17167%3A58131",
+    },
+  }}
+  argTypes={{
+    blockName: { table: { disable: true } },
+    className: { table: { disable: true } },
+    modifiers: { table: { disable: true } },
+    id: { table: { disable: true } },
+    layout: { table: { disable: false } },
+    imageAspectRatio: { table: { disable: false } },
+    imageFlip: { table: { disable: false } },
+    border: { table: { disable: false } },
+    padding: { table: { disable: false } },
+    backgroundColor: { control: { type: "color" } },
+    foregroundColor: { control: { type: "color" } },
+  }}
+/>
+
+# Card
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.24.0`   |
+
+<Description of={Card} />
+
+Cards are contained units of information that can stand independently from other elements for context. While a Card is a self-contained unit, Cards are often grouped together and displayed in some type of grid layout.
+
+<Preview withToolbar>
+  <Story
+    name="Card Props"
+    args={{
+      id: "custom-card",
+    }}
+  >
+    {(args) => (
+      <Card {...args}>
+        <CardImage
+          src="https://placeimg.com/400/300/animals"
+          alt="Alt text"
+          imageAspectRatio={args.imageAspectRatio}
+        />
+        <CardHeading level={2} id="heading1">
+          Optional Header
+        </CardHeading>
+        <CardHeading level={4} id="heading2">
+          Sollicitudin Lorem Tortor Purus Ornare
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue. Nullam quis risus eget urna mollis ornare
+          vel eu leo. Nulla vitae elit libero, a pharetra augue.
+        </CardContent>
+        <CardActions topBorder bottomBorder>
+          <Button
+            onClick={action("clicked")}
+            id="button1"
+            buttonType={ButtonTypes.Primary}
+            type="submit"
+          >
+            {"Primary"}
+          </Button>
+          <Button
+            onClick={action("clicked")}
+            id="button2"
+            buttonType={ButtonTypes.Secondary}
+            type="submit"
+          >
+            {"Secondary"}
+          </Button>
+        </CardActions>
+        <CardActions>
+          <Link onClick={action("clicked")} href="blah">
+            {"#hash1"}
+          </Link>
+          ,<Link onClick={action("clicked")} href="blah">
+            {"#hash2"}
+          </Link>,<Link onClick={action("clicked")} href="blah">
+            {"#hash3"}
+          </Link>
+        </CardActions>
+      </Card>
+    )}
+  </Story>
+</Preview>
+
+<ArgsTable story="Card Props" />
+
+## Image Position
+
+### Vertical Cards
+
```

Comment:
It's not obvious that the `imageFlip` attribute is what makes the image render differently in the examples. It'll be good to have a short description here for that.

Reply:
The image controls have been moved into props, so this shoudl now be less confusing.  I also renamed this prop to `imageAtEnd`.